### DISCUSSION TITLE
Usage of input templates

### DISCUSSION THREAD

**Discussion by jacobholm** [BryanJenksCommunity/FAQ#179] [Q&A] [answered]
Hi!

First time poster, so first of all, thank you for all the content 👍 

I was wondering how you use the input templates, e.g. "Book". Do you start with a completely new seedling note (using the "New" template) and then migrate to a Book note down the road? Or perhaps just create a Book note, fill it out and then link to an existing seedling/evergreen/etc note?

In the Book note I see there are already bulletpoints for notes and commens.

Thanks.

**Comment by tallguyjenks** [comment, marked as answer]
1. i create a new note for a book I've finished or are currently reading
2. Insert my `meta` template which sees the symbol at the start of the file name a `{`
3. `meta` inserts the `book` template 
4. then i create my notes and link them to the book note where applicable
